### ISSUE TITLE
Unable to find template plugin matching 'plugin-typescript-swr'

### ISSUE THREAD

**Issue by DaanKorver** [croutonn/graphql-codegen-plugin-typescript-swr#223]
- **I'm submitting a ...**
  [x] bug report
  [ ] feature request
  [ ] question about the decisions made in the repository
  [ ] question about how to use this project

- **Summary**
I am getting the following error:
```bash
      Unable to find template plugin matching 'plugin-typescript-swr'
      Install one of the following packages:
      - @graphql-codegen/plugin-typescript-swr
      - @graphql-codegen/plugin-typescript-swr-template
      - @graphql-codegen/plugin-typescript-swr-plugin
      - graphql-codegen-plugin-typescript-swr
      - graphql-codegen-plugin-typescript-swr-template
      - graphql-codegen-plugin-typescript-swr-plugin
      - codegen-plugin-typescript-swr
      - codegen-plugin-typescript-swr-template
      - plugin-typescript-swr
```

Even though I have it installed, still getting this error. This is my pacakge.json
```json
-- Rest of package.json
...
  "dependencies": {
    "@craco/craco": "^6.2.0",
    "@emailjs/browser": "^3.7.0",
    "@google-cloud/dialogflow": "^4.3.1",
    "@google-cloud/storage": "^5.14.1",
    "@heroicons/react": "^1.0.4",
    "@react-hook/media-query": "^1.1.1",
    "@tailwindcss/line-clamp": "^0.2.1",
    "@tailwindui/react": "^0.1.1",
    "@vercel/fetch": "^6.1.1",
    "@vercel/node": "^1.12.1",
    "autoprefixer": "^9",
    "classnames": "^2.3.1",
    "craco": "^0.0.3",
    "dotenv": "^16.0.3",
    "email-validator": "^2.0.4",
    "googleapis": "^86.1.0",
    "graphql": "^16.6.0",
    "graphql-request": "^5.2.0",
    "html2canvas": "^1.3.2",
    "imtool": "^1.1.4",
    "moment": "^2.29.1",
    "postcss": "^8.4.21",
    "prettier": "^2.3.2",
    "puppeteer": "^19.8.3",
    "react": "^17.0.2",
    "react-component-export-image": "^1.0.6",
    "react-dom": "^17.0.2",
    "react-dropzone": "^11.3.4",
    "react-google-autocomplete": "^2.4.2",
    "react-router-dom": "^5.2.1",
    "react-scripts": "^5.0.1",
    "react-slick": "^0.28.1",
    "react-to-print": "^2.14.0",
    "react-webcam": "^6.0.0",
    "shopify-buy": "^2.11.0",
    "slick-carousel": "^1.8.1",
    "smoothscroll-polyfill": "^0.4.4",
    "swiper": "^6.0.0",
    "swr": "^2.1.3",
    "tailwindcss": "^3.3.1",
    "three": "^0.151.2",
    "use-persisted-state": "^0.3.3",
    "uuid": "^8.3.2"
  },
  "devDependencies": {
    "@graphql-codegen/cli": "^3.3.0",
    "@graphql-codegen/client-preset": "3.0.0",
    "@graphql-codegen/introspection": "3.0.1",
    "@graphql-codegen/schema-ast": "^3.0.0",
    "@graphql-codegen/typescript": "^3.0.3",
    "@graphql-codegen/typescript-graphql-request": "^4.5.9",
    "@graphql-codegen/typescript-operations": "^3.0.3",
    "@types/google.maps": "^3.52.3",
    "@types/node": "^18.15.7",
    "@types/react": "^18.0.28",
    "@types/react-dom": "^18.0.11",
    "@types/react-router-dom": "^5.3.3",
    "@types/react-slick": "^0.23.10",
    "@types/shopify-buy": "^2.17.0",
    "@types/three": "^0.150.1",
    "@types/use-persisted-state": "^0.3.1",
    "@types/uuid": "^9.0.1",
    "@types/webfontloader": "^1.6.35",
    "graphql-codegen-plugin-typescript-swr": "^0.8.4",
    "prettier": "^2.8.7",
    "typescript": "^5.0.4"
  }
...
// Rest of package.json
```

- **Other information** (e.g. detailed explanation, stack traces, related issues, suggestions how to fix, links for us to have context, eg. StackOverflow, personal fork, etc.)

Ive tried cleaning my yarn cache, removing the node_modules and yarn.lock file. Also tried reinstalling graphql-codegen/typescript package.

**Comment by mrkutly** [comment]
Just opened an issue for the same problem [here](https://github.com/croutonn/graphql-codegen-plugin-typescript-swr/issues/224). It appears the `build` directory is missing in the latest release.

**Comment by eitoball** [comment]
@DaanKorver Just released v0.8.5 to fix packaging issue. Please try, and let me know how it goes.
